src/components/Tooltip/index.tsx

```diff
@@ -25,6 +25,7 @@ interface TooltipContentProps extends Omit<PopoverProps, 'content'> {
   onOpen?: () => void
   // whether to wrap the content in a `TooltipContainer`
   wrap?: boolean
+  disableHover?: boolean // disable the hover and content display
```

Comment:
have you considered doing this via `content` instead of a new prop? i.e. caller would handle the disabled state and send `content=null` when it's disabled. 

not saying it's right approach, just wondering